Core/Validation/EmailConfirmationRequestValidation.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Core.DTO.EmailDTO;
+using FluentValidation;
+
+namespace Core.Validation
+{
+    public class EmailConfirmationRequestValidation : AbstractValidator<EmailConfirmationTokenRequestDTO>
+    {
+        public EmailConfirmationRequestValidation()
+        {
+            RuleFor(x => x.Token)
+                .NotEmpty()
+                .NotNull()
+                .WithMessage("Token is null !");
```

Comment:
FluentValidation has its messages on NotEmpty() and NotNull(), you don't need to write your own